flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/ResourceActions.java

```diff
@@ -52,4 +52,6 @@
 	 * @param cause of the allocation failure
 	 */
 	void notifyAllocationFailure(JobID jobId, AllocationID allocationId, Exception cause);
+
+	void cancelResourceAllocation(ResourceProfile resourceProfile);
```

Comment:
I think we don't necessarily need this call. See my comment in the `SlotManager`.

Reply:
I comment it is slot manager.